### PR TITLE
Updated docker compose files, .gitmodule for new service simulation

### PR DESCRIPTION
**Pull request by KiLanger**

### CODE REVIEW

**Comment by KiLanger** [comment]
> you should also adapt the MISARCH_SHIPMENT_PROVIDER_ENDPOINT env variable for shipment also, does payment have sth similar?

For payment, its defined on the service level (in the `docker-compose-base.yaml`)
In my opinion, that makes sense, since there are already multiple service level variables (will be changeable via experiment-config service)

File: docker-compose.yaml [hunk 1]
```diff
@@ -294,6 +294,18 @@ services:
...
+    extends:
+      file: simulation/docker-compose-base.yaml
+      service: simulation
+    image: ghcr.io/misarch/simulation:main
+  simulation-rabbitmq:
+    extends:
+      file: simulation/docker-compose-base.yaml
+      service: simulation-rabbitmq
+    ports:
+      - "15672:15672"
```
[the remark this answers is not in the document]
Comment (KiLanger) [reply docker-compose.yaml:307]: It makes the simulation queueing more transparent.
We also expose dar-redis which is a similar context in my opinion.

**Review by nk-coding — CHANGES_REQUESTED** [review]
env variable(s) to configure endpoints still missing

File: docker-compose.yaml [hunk 2]
```diff
@@ -299,12 +299,12 @@ services:
       file: simulation/docker-compose-base.yaml
       service: simulation
     image: ghcr.io/misarch/simulation:main
+    ports:
+      - "7000:8080"
```
[the remark this answers is not in the document]
Comment (KiLanger) [reply docker-compose.yaml:305]: This is the simulation endpoint ? :D
Required to manually update payments ands shipments

[the remark this answers is not in the document]
Comment (nk-coding) [reply docker-compose.yaml:305]: Oooh, I totally forgot about this feature again

**Review by nk-coding — CHANGES_REQUESTED** [review]
isn't the dev docker compose file still missing the env variables?

**Review by nk-coding — APPROVED** [review]

**Pull request merged**